packages/aws-cdk-lib/aws-cognito/lib/user-pool.ts

```diff
@@ -767,6 +767,12 @@ export interface IUserPool extends IResource {
    */
   readonly userPoolArn: string;
 
+  /**
+   * The provider name of this user pool resource
+   * @attribute
```

Comment:
nit: add a line break
```suggestion
   * The provider name of this user pool resource
   *
   * @attribute
```